modules/testsuite/cxf-tests/pom.xml

```diff
@@ -498,6 +498,272 @@
+                                      <script>file:///${basedir}/src/test/scripts/jbws-testsuite-default-picketbox.groovy</script>
+                                  </scripts>
+                                  <properties>
+                                      <property>
+                                          <name>inputFile</name>
+                                          <value>${jboss.home}/standalone/configuration/standalone.xml</value>
+                                      </property>
+                                      <property>
+                                          <name>outputFile</name>
+                                          <value>${jboss.home}/standalone/configuration/jbws-testsuite-default.xml</value>
+                                      </property>
+                                      <property>
+                                          <name>serverLog</name>
+                                          <value>jbws-testsuite-default.log</value>
+                                      </property>
+                                      <property>
+                                          <name>usersPropFile</name>
+                                          <value>${project.build.directory}/test-resources/jaxws/samples/wsse/policy/jaas/digest/WEB-INF/jbossws-users.properties</value>
+                                      </property>
+                                      <property>
+                                          <name>rolesPropFile</name>
+                                          <value>${project.build.directory}/test-resources/jaxws/samples/wsse/policy/jaas/digest/WEB-INF/jbossws-roles.properties</value>
+                                      </property>
+                                      <property>
+                                          <name>keystorePath</name>
+                                          <value>${project.build.directory}/test-classes/test.keystore</value>
+                                      </property>
+                                      <property>
+                                          <name>testResourcesDir</name>
+                                          <value>${project.build.directory}/test-resources</value>
+                                      </property>
+                                  </properties>
+                              </configuration>
+                          </execution>
+                          <execution>
+                              <id>testsuite-ssl-mutual-auth</id>
+                              <phase>pre-integration-test</phase>
+                              <goals>
+                                  <goal>execute</goal>
+                              </goals>
+                              <configuration>
+                                  <scripts>
+                                      <script>file:///${basedir}/src/test/scripts/jbws-testsuite-ssl-mutual-auth-elytron.groovy</script>
```

Comment:
There were no changes needed to the server configuration related to picketbox
for this category of tests.  I chose to maintain the reference to the original 
file and not make a duplicate with just a name change.

The only reason I maintained the "-elytron" designator from the (original) files 
was to relate back your original naming scheme.  I felt this would provide clarity 
to the user community that was familiar with the old naming convention.

Reply:
@rsearls  But this groovy script is to create the tls element under elytron subsystem for https, and it all uses the elytron configuration instead of picketbox.